### PR TITLE
feat: add Real-Time Counter Lab

### LINKED ISSUE #454: chore: add change event lab to frontend cert
We have a workshop and lecture videos to teach campers about the DOM and change events.

Here is the preceding workshop
https://github.com/freeCodeCamp/CurriculumExpansion/pull/453

Campers will have already complete the previous fundamentals modules(strings, variables, arrays, numbers, Math object, objects, etc) So you can see the sheet for more details to see the complete list of what is covered.

For this lab, we want them to practice change events and adding HTML using `innerHTML`.

## Acceptance criteria

- [ ] create a new folder of the project in the `frontend-cert/js-projects` directory
- [ ] add your completed project in an `index.html`, `styles.css` and `script.js` files in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #454

### CODE REVIEW

File: frontend-cert/js-projects/lab-realtime-character-counter/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+You should have an `h1` element with the text `Real-Time Character Counter`.
+
+You should have a `textarea` element with the `id` of `textInput`. It should have a `placeholder` attribute with the text `Type something...`.
+
+There should be a `p` `element` with the `id` of `charCount`. It should initially contain the text `Character Count: 0/50`. This placeholder text should be replaced with the current count of characters in the `textarea` element.
```
Comment (Dario-DC) [inline frontend-cert/js-projects/lab-realtime-character-counter/user-stories.md:5]:
```suggestion
There should be a `p` element with the `id` of `charCount`. It should initially contain the text `Character Count: 0/50`. This placeholder text should be replaced with the current count of characters in the `textarea` element.
```

**Review by Dario-DC — APPROVED** [review]
Left a comment. LGTM :+1: 
Is this JS only in the sense that the HTML and CSS are provided?

**Comment by zairahira** [comment]
Sorry for adding a confusing title, the lab wasn't JS only.
I have updated it.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**